torch/csrc/deploy/example/generate_examples.py

```diff
@@ -0,0 +1,43 @@
+"""
+Generate the example files that torchpy_test uses.
+"""
+from pathlib import Path
+import torch
+import argparse
+
+from torch.package import PackageExporter
+
+try:
+    from .examples import Simple, resnet18
+except ImportError:
+    from examples import Simple, resnet18
+
+def save(name, model, model_jit, eg):
+    with PackageExporter(str(p / name)) as e:
+        e.mock('iopath.**')
+        e.save_pickle('model', 'model.pkl', model)
+        e.save_pickle('model', 'example.pkl', eg)
+    model_jit.save(str(p / (name + '_jit')))
+
+
+parser = argparse.ArgumentParser(description="Generate Examples")
+parser.add_argument("--install_dir", help="Root directory for all output files")
+parser.add_argument("--fbcode_dir", help="Root directory for all output files")
```

Comment:
'help' is wrong copypasta - confirm if 'fbcode_dir' is even really needed? it gives access to read from files in the fbcode tree without specifying them as inputs to the target, but doesn't look like it's being used

Reply:
the fbcode custom_rule for python objects drops this argument in, so if it isn't here then it just fails...

Reply:
oh, i think if you add 'strict=true' to the custom_rule then it doesn't.  I should say i know this behavior is true but i forgot the flag name.